### PR TITLE
Create notebook that prepares the dataset and updates README.md

### PR DESCRIPTION
**Pull request by santisoler**
Add notebook that prepares a dataset that contains observed gravity data and
precomputed gravity fields (gravity disturbance, Bouguer gravity disturbance)
over Bushveld, Southern Africa. Observation heights are provided as referenced
on mean sea-level and referenced on the ellipsoid. Edit the `environment.yml`
file with required packages. Generate a `preview.jpg` figure. Edit the
`README.md` with information about the dataset.

### CODE REVIEW

**Comment by santisoler** [comment]
Instead of waiting until next Ensaio release for the Southern Africa topography grid, the `prepare.ipynb` notebook downloads it directly from the Zenodo archive.

File: README.md [hunk 1]
```diff
@@ -1,39 +1,46 @@
-<!--
-This is a template for datasets in the Fatiando a Terra collection.
+# Bushveld, Southern Africa - Observed and preprocessed gravity
 
-When adding a new dataset, fill out the information below. Search for all
-instances of "CHANGEME" and replace with the relevant information.
+This gravity contains ground gravity observations over the area that comprises
```
Comment (leouieda) [inline README.md]:
```suggestion
This dataset contains ground gravity observations over the area that comprises
```

File: README.md [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
 | License | [CC-BY](https://creativecommons.org/licenses/by/4.0/) |
-| MD5 | `md5:CHANGEME` |
-| SHA256 | `sha256:CHANGEME` |
-| Source | CHANGEME: Citation and/or link to original data source |
-| Original license | CHANGEME: License (with link) of the original source |
+| MD5 | `md5:368284cc210c6bbe256e9e49e892f262` |
+| SHA256 | `sha256:3fc1daf74a2fc3bcc3cf7f72a632518f8c6b6e306ce12fddf4055d7cb44945c8` |
+| Source | [NOAA NCEI](https://www.ngdc.noaa.gov/mgg/gravity/) (gravity) and [ETOPO1](https://doi.org/10.7289/V5C8276M) (topography) |
+| Original license | [public domain](https://ngdc.noaa.gov/ngdcinfo/privacy.html) (gravity) and [public domain](https://ngdc.noaa.gov/mgg/global/dem_faq.html#sec-2.4) (topography) |
 | Processing code | [`prepare.ipynb`](https://nbviewer.org/github/fatiando-data/CHANGEME/blob/main/prepare.ipynb) |
```
Comment (leouieda) [inline README.md]:
```suggestion
| Processing code | [`prepare.ipynb`](https://nbviewer.org/github/fatiando-data/bushveld-gravity/blob/main/prepare.ipynb) |
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
-* CHANGEME: List here changes made to the original data.
+* The original data were cropped to a region bounded by 25 and 32 degrees on
+  longitude and -27 and -23 degrees on latitude.
+* Geometric observation heights were obtained by adding geoid heights to the
+  original observation heights referenced on the mean sea-level. The geoid
+  heights on each observation point were obtained by interpolation of the geoid
+  available in doi:
+  [10.5281/zenodo.5882205](https://doi.org/10.5281/zenodo.5882205).
+* Gravity disturbances were computed by removing the normal gravity of the
+  WGS84 computed through [Boule](https://www.fatiando.org/boule).
```
Comment (leouieda) [inline README.md]:
```suggestion
  WGS84 ellipsoid computed through [Boule](https://www.fatiando.org/boule).
```

File: README.md [hunk 4]
```diff
@@ -1,39 +1,46 @@
...
+* Bouguer gravity disturbances were computed by forward modelling the
+  topography using [Harmonica](https://www.fatiando.org/harmonica) starting
+  from the topography grid provided in doi:
+  [10.5281/zenodo.6481379](https://doi.org/10.5281/zenodo.6481379).
```
Comment (leouieda) [inline README.md]:
```suggestion
  from the topography grid provided in doi:
  [10.5281/zenodo.6481379](https://doi.org/10.5281/zenodo.6481379)
  and using densities of 2670 kg/m³ above the ellipsoid and
  1040 - 2670 kg/m³ below the ellipsoid.
```

**Review by leouieda — APPROVED** [review]
A few corrections to the README text but nothing major. Notebook looks great! Feel free to commit and then merge this 👍🏾

**Comment by santisoler** [comment]
Thanks for the review @leouieda . Good catches!

I'm merging this and then I'll make the release following the checklist.

**Pull request merged**